app/controllers/tweet_controller.rb

```diff
@@ -0,0 +1,70 @@
+class TweetController < ApplicationController
+  get '/tweets' do
+    if logged_in?
+      @all_tweets = Tweet.all
+      erb :'tweets/tweets'
+    else
+      redirect '/login'
+    end
+  end
+  
+  get '/tweets/new' do
+    if logged_in?
+      current_user
+      erb :'tweets/create_tweet'
+    else
+      redirect '/login'
+    end
+  end
+  
+  post '/tweets' do
+    if params[:content] != "" 
+      Tweet.create(content: params[:content], user_id: current_user.id)
+    else
+      redirect '/tweets/new'
+    end
+  end
+  
+  get '/tweets/:tweet_id' do
+    if logged_in?
+      @tweet = Tweet.find_by(id: params[:tweet_id])
+    else
+      redirect '/login'
+    end
+    erb :'tweets/show_tweet'
+  end
+  
+  get '/tweets/:tweet_id/edit' do
+    if logged_in?
```

Comment:
Shouldn't you verify a tweet belongs to the `current_user` (and not just that any user is logged in) before allowing them to edit?